akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -515,16 +513,33 @@ object Sink {
    * will be sent to the destination actor.
    * When the stream is completed with failure - result of `onFailureMessage(throwable)`
    * function will be sent to the destination actor.
-   *
    */
   def actorRefWithAck[T](
       ref: ActorRef,
       onInitMessage: Any,
       ackMessage: Any,
       onCompleteMessage: Any,
-      onFailureMessage: (Throwable) => Any = Status.Failure): Sink[T, NotUsed] =
+      onFailureMessage: Throwable => Any): Sink[T, NotUsed] =
```

Comment:
The default param creates a different signature when compiled with Scala, so I think this isn't bincomp safe, even if it is source comp safe. That's the reason why MiMa complains.

Reply:
I've to revert this part then. I don't see a way out of this until there is a binary breaking version.